utils/firestore.js

```diff
@@ -417,7 +417,8 @@ const removeParticipantsDataDestruction = async () => {
         let count = 0;
         const millisecondsWait = 5184000000; // 60days
         // Stub records that will be retained after deleting data.
-        const stubFieldArray = [ "query", "pin", "token", "state", "Connect_ID", "471168198", "736251808", "436680969", "480305327", "564964481", "795827569", "544150384", "371067537", "454205108", "454445267", "919254129", "412000022", "558435199", "262613359", "821247024", "914594314", "747006172", "659990606", "299274441", "919699172", "141450621", "576083042", "431428747", "121430614", "523768810", "639172801", "175732191", "150818546", "624030581", "285488731", "596510649", "866089092", "990579614", "131458944", "372303208", "777719027", "620696506", "352891568", "958588520", "875010152", "404289911", "637147033", "734828170", "715390138", "538619788", "153713899", "613641698", "407743866", "831041022", "269050420", "359404406", "119449326", "304438543", "912301837", "130371375", "765336427", "479278368", "826240317", "693626233", "104278817", "744604255", "268665918", "592227431", "399159511", "231676651", "996038075", "506826178", "524352591", "902332801", "457532784", "773707518", "577794331", "883668444", "827220437", "699625233", ];
+        const stubFieldArray = [ "query", "pin", "token", "state", "Connect_ID", "471168198", "736251808", "436680969", "480305327", "564964481", "795827569", "544150384", "371067537", "454205108", "454445267", "919254129", "412000022", "558435199", "262613359", "821247024", "914594314", "747006172", "659990606", "299274441", "919699172", "141450621", "576083042", "431428747", "121430614", "523768810", "639172801", "175732191", "150818546", "624030581", "285488731", "596510649", "866089092", "990579614", "131458944", "372303208", "777719027", "620696506", "352891568", "958588520", "875010152", "404289911", "637147033", "734828170", "715390138", "538619788", "153713899", "613641698", "407743866", "831041022", "269050420", "359404406", "119449326", "304438543", "912301837", "130371375", "765336427", "479278368", "826240317", "693626233", "104278817", "744604255", "268665918", "592227431", "399159511", "231676651", "996038075", "506826178", "524352591", "902332801", "457532784", "773707518", "577794331", "883668444", "827220437", "699625233", "861639549", "731498909", "787567527", "222373868", "648936790", "297462035", "648228701", "438636757", "320023644", "265193023", "222161762", "253883960", "764863765", "547363263", "195145666", "459098666", "217640691", "220186468", "784810139", "320303124", "843688458", "949302066", "517311251", "536735468", "832139544", "976570371", "770257102", "663265240", "264644252", "126331570", "315032037", "173836415"];
```

Comment:
This looks fine but I have one small request... Can you please update this to make it more readable? It's hard to know at a glance what is actually in this stub record. My first thought is to put all of these fields in the `fieldToConceptIdMapping.js` file and then reference them from there.

Reply:
Solved